spec/intl.html

```diff
@@ -177,8 +178,10 @@ <h1>InitializeDateTimeFormat ( _dateTimeFormat_, _locales_, _options_ )</h1>
           1. <del>Set _dateTimeFormat_.[[HourCycle]] to _hc_.</del>
           1. <del>If _dateTimeformat_.[[HourCycle]] is *"h11"* or *"h12"*, then</del>
             1. <del>Let _pattern_ be _bestFormat_.[[pattern12]].</del>
+            1. <del>Let _rangePatterns_ be _bestFormat_.[[rangePatterns12]].</del>
           1. <del>Else,</del>
             1. <del>Let _pattern_ be _bestFormat_.[[pattern]].</del>
+            1. <del>Let _rangePatterns_ be _bestFormat_.[[rangePatterns]].</del>
         1. <del>Else,</del>
           1. <del>Set _dateTimeFormat_.[[HourCycle]] to *undefined*.</del>
           1. <del>Let _pattern_ be _bestFormat_.[[pattern]].</del>
```

Comment:
The formatting and indentation relative to upstream is confusing here; anyway, this Else block doesn't have anything corresponding in the upstream to delete, and it omits `_rangePatterns_`. Are they out of sync? Since you're fixing the _rangePatterns_ elsewhere, should sync this block as well.

Reply:
Yes, the ECMA-402 base of Temporal is out of sync, and is older than the ECMA-402 base of `formatRange`. Let's rebase Temporal on top of ECMA-402 master.